types/indefinite/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for indefinite 2.2
+// Type definitions for indefinite 2.3
 // Project: https://github.com/tandrewnichols/indefinite
-// Definitions by: omaishar <https://github.com/omaishr>
+// Definitions by: omaishar <https://github.com/omaishar>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 export interface Options {
+    articleOnly?: boolean;
     capitalize?: boolean;
     caseInsensitive?: boolean;
     numbers?: "colloquial";
 }
-export function indefinite(word: string | number, opts?: Options): string;
+export default function(word: string | number, opts?: Options): string;
```

Comment:
You actually want to use `export =` instead since the underlying library doesn't use default exports. See [this](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) for more details.

Reply:
Hopefully https://github.com/DefinitelyTyped/DefinitelyTyped/pull/40238/commits/e20feba31ebc4126246767b95dd03db65b9850a2 is right!